web/src/components/storage/ProposalSpacePolicySection.jsx

```diff
@@ -447,7 +447,12 @@ export default function ProposalSpacePolicySection({
   const currentPolicy = SPACE_POLICIES.find(p => p.id === settings.spacePolicy) || SPACE_POLICIES[0];
 
   return (
-    <Section title={_("Find Space")}>
+    <Section
+      // TRANSLATORS: The storage "Find Space" section's title
+      title={_("Find Space")}
+      // TRANSLATORS: The storage "Find space" sections's description
+      description={_("Allocating the file systems might need to find free space in the devices listed below. Choose how to do it")}
```

Comment:
NP: having the setence without dot at the end looks strange to me.

Reply:
I just followed the [@ancorgs's comment](https://github.com/openSUSE/agama/pull/1045#issuecomment-1959066761), which does not includes the dot at the end of these sentences. Should we add them?

Reply:
The comment was not intended to be taken so literally. I have nothing against dots. Some of my best friends are dots.